scripts/compile.sh

```diff
@@ -39,19 +46,9 @@ unset DOTNET_TOOLS
 
 DOTNET_PATH=$(which dotnet)
 PREFIX="$(cd -P "$(dirname "$DOTNET_PATH")/.." && pwd)"
-DNX_ROOT="$PREFIX/share/dotnet/cli/bin/dnx"
-if [ ! -d "$DNX_ROOT" ]; then
-    echo "warning: your stage0 is using the old layout" 1>&2
-    DNX_ROOT="$PREFIX/share/dotnet/cli/dnx"
-fi
-
-if [ ! -d "$DNX_ROOT" ] || [ ! -e "$DNX_ROOT/dnx" ]; then
-    echo "error: could not find DNX in $DNX_ROOT" 1>&2
-    exit 1
-fi
 
 header "Restoring packages"
-dotnet restore "$REPOROOT" --quiet --runtime "osx.10.10-x64" --runtime "ubuntu.14.04-x64" --runtime "win7-x64" --no-cache
+$DNX_ROOT/dnu restore "$REPOROOT" --quiet --runtime "osx.10.10-x64" --runtime "ubuntu.14.04-x64" --runtime "win7-x64" --no-cache
```

Comment:
The version of dnu we had in our LKG was bugging out on restoring the StdLibrary. To upgrade we need to do this at least once where we use dnu instead of dotnet restore. We could remove it after the LKG version has the newer dnu